tests/test_backbones/test_tcn.py

```diff
@@ -57,3 +82,17 @@ def test_tcn_backbone():
     assert feat[1].shape == (2, 1024, 217)
     assert feat[2].shape == (2, 1024, 163)
     assert feat[3].shape == (2, 1024, 1)
+
+    # Test TCN with 4 blocks (train_with_stride_conv == True)
+    model = TCN(
+        in_channels=34,
+        num_blocks=4,
+        kernel_sizes=(3, 3, 3, 3, 3),
+        train_with_stride_conv=True)
+    pose2d = torch.rand((2, 34, 243))
+    feat = model(pose2d)
+    assert len(feat) == 4
+    assert feat[0].shape == (2, 1024, 27)
+    assert feat[1].shape == (2, 1024, 9)
+    assert feat[2].shape == (2, 1024, 3)
+    assert feat[3].shape == (2, 1024, 1)
```

Comment:
Could you please add a test in the unittest to check the model w. or w/o train_with_stride_conv will have the same output and gradient after a forward+backward pass?

Reply:
Will add a test.